aggregators/moki/index.ts

```diff
@@ -0,0 +1,31 @@
+import {Fetch, SimpleAdapter} from "../../adapters/types";
+import {CHAIN} from "../../helpers/chains";
+import fetchURL from "../../utils/fetchURL";
+
+const URL = 'https://api.leapwallet.io/';
+const endpoint = 'ton-sor/api/v1/analytics/volumes';
+const start = 1727711103;// 30.09.2024
+
+interface IAPIResponse {
+    dailyVolume: string;
+    totalVolume: string;
+}
+
+const fetch: Fetch = async (timestamp) => {
+    const {dailyVolume, totalVolume}: IAPIResponse = await fetchURL(`${URL}${endpoint}?timestamp=${timestamp * 1000}`);
```

Comment:
@Ishnoor04 thanks for the PR, but is this the start timestamp of the 24 range or end timestamp? also, the api is not working?

Reply:
@g1nt0ki this is the 24 hour range end timestamp. Api is working now.

Reply:
thanks for the PR